dlp/risk_test.py

```diff
@@ -12,18 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from gcp_devrel.testing.flaky import flaky
+from flaky import flaky
 import google.cloud.pubsub
 
 import pytest
+import os
 
 import risk
 
-GCLOUD_PROJECT = "python-docs-samples"
-TABLE_PROJECT = "python-docs-samples"
+GCLOUD_PROJECT = os.environ.get('GCLOUD_PROJECT')
```

Comment:
I was confused about how these variables were getting assigned in another PR. Thank you for fixing this!